src/sentry/tasks/reports.py

```diff
@@ -438,6 +492,94 @@ def build_project_calendar_series(interval, project):
     return clean_calendar_data(project, series, start, stop, rollup)
 
 
+def build_key_events(interval, project):
+    start, stop = interval
+
+    # Take the 3 most frequently occuring events
```

Comment:
When you say "events" here you are referring to errors, right?

Reply:
yeah lol. changed the name to `build_key_errors`